diff --git a/lib/crypto-node.js b/lib/crypto-node.js
index 1e11f2e2bfe51708d809c038186d996ba06c9cb1..6042f881d29812426e439380f8f71bad5073f87c 100644
--- a/lib/crypto-node.js
+++ b/lib/crypto-node.js
@@ -1,5 +1,3 @@
-// this can be removed once we only support Node 20+
-export * from "node:crypto";
 import { createPrivateKey } from "node:crypto";
 
 import { isPkcs1 } from "./utils.js";
diff --git a/lib/get-token.js b/lib/get-token.js
index a0d04e4ec5370924e512e759724c67eae62b2baf..83fad2f96ff7e2512fada93fd6b153459352ad10 100644
--- a/lib/get-token.js
+++ b/lib/get-token.js
@@ -9,7 +9,8 @@ import {
   base64encode,
 } from "./utils.js";
 
-import { subtle, convertPrivateKey } from "#crypto";
+import { convertPrivateKey } from "#crypto";
+import { subtle } from "node:crypto";
 
 /**
  * @param {import('../internals').GetTokenOptions} options
